x/stakeibc/keeper/icqcallbacks_withdrawal_balance.go

```diff
@@ -62,32 +58,15 @@ func WithdrawalBalanceCallback(k Keeper, ctx sdk.Context, args []byte, query icq
 		return errorsmod.Wrapf(types.ErrICAAccountNotFound, "no fee account found for %s", chainId)
 	}
 
-	// Determine the stride commission rate to the relevant portion can be sent to the fee account
-	params := k.GetParams(ctx)
-	strideCommissionInt, err := cast.ToInt64E(params.StrideCommission)
+	// Split the withdrawal amount into a rebate, stride fee, and reinvest portion
```

Comment:
It seems like some of the sanity checks from `WithdrawalBalanceCallback` didn't get ported over to `CheckForCommunityPoolRebate` - should we port those over? e.g. sanity checks on commission (0 <= x <= 1)

Reply:
commission is validated by the params. Imo the other check falls into my category above of unnecessarily validating simple math (i.e. reinvest = balance - fee, then validating fee + balance = reinvest). Wdyt?